virttest/qemu_vm.py

```diff
@@ -1884,7 +1884,8 @@ def sort_key(dev):
             add_virtio_rng(devices, rng_params, parent_bus)
 
         # Add logging
-        devices.insert(StrDev('isa-log', cmdline=add_log_seabios(devices)))
+        if params.get("enable_debugcon") == "yes":
+            devices.insert(StrDev('isa-log', cmdline=add_log_seabios(devices)))
```

Comment:
This for instance is an option we never used so far so I am glad we can now disable it if we don't need it.